### DISCUSSION TITLE
Aqara(Xiaomi) E1 Door & Window Sensor (MCCGQ14LM) not working

### DISCUSSION THREAD

**Discussion by f45tb00t** [arendst/Tasmota#13848] [Q&A] [answered]
### PROBLEM DESCRIPTION
Hello,

I am using Tasmota v10.0.0(zbbridge) since today. I was with v9.4.0 before. Unfortunately the new Aqara devices are not working.

It seems the firmware needs to be upgraded.

Here are some references:
https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5403
https://github.com/Koenkk/zigbee2mqtt/issues/9274
https://github.com/home-assistant/core/issues/58985

@theo: Is there anything you can do?

Kind regards

fb

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Sonoff Bridge
- [x] Tasmota binary firmware version number used: 10.0.0
  - [x] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
- Just use the E1

### EXPECTED BEHAVIOUR
- Battery Status
-  Open/Closed Status

### SCREENSHOTS
None

### ADDITIONAL CONTEXT
None

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by namirda** [comment]
I would also like this new and popular device to be supported by zigbee2tasmota. 

The OP provided links to several threads where the problem with this device have been discussed and the latest version of Zigbee2mqtt (1.22.1) now claims to support the device.

It is not at all clear to me how the fix which has been made in Zigbee2mqtt can be ported to zigbee2tasmota.

Can anybody explain?

I presume a new version of the ncp-uart-nsw firmware required? Is this related to this discussion https://github.com/arendst/Tasmota/discussions/13941 ??

Thanks

**Comment by CrazyMan2000** [comment]
Same here.
I just bought 4 of these sensors and totally missed they werent supported yet.

**Comment by f45tb00t** [comment]
Bump. anyone?

**Comment by f45tb00t** [comment, marked as answer]
Hi guys,

seems I have good news :)

> Pairing Instructions
> Press and hold the button on the device until the blue light starts blinking, release it and the pairing should begin.
> 
> If you're having problems pairing, try keeping the device alive (press the button every second) after pairing started, until pairing completed (source: ZHA with Zigate and Aqara device). 
Ref: https://zigbee.blakadder.com/Aqara_MCCGQ14LM.html

I've just tested it with one of my devices. Basically I can see now Power and also the detection open/close works. :)


Cheers

fb

**Comment by namirda** [comment]
But what has changed?

I have tried with 4 different devices.

It didn't work before and it still doesn't work now.............

**Reply by sfromis** [reply]
What Tasmota version are you using? Maybe try updating to most recent....

**Reply by namirda** [reply]
I am using 10.1.0 which I think is the latest. 

The devices seem to pair but do not give Contact information.  No response to the magnet.

Pressing the button on the devices after pairing shows the following on the console:

23:52:55.413 MQT: tele/tasmota_zigbee/SENSOR = {"ZbReceived":{"0xF39E":{"Device":"0xF39E","ModelId":"lumi.magnet.acn001","Endpoint":1,"LinkQuality":60}}}
23:52:56.283 MQT: tele/tasmota_zigbee/SENSOR = {"ZbReceived":{"0xF39E":{"Device":"0xF39E","ModelId":"lumi.magnet.acn001","Endpoint":1,"LinkQuality":60}}}

**Reply by sfromis** [reply]
Using the magnet with a similar door/window sensor `lumi.sensor_magnet.aq2` I'm getting
`{"xxxx":{"Device":"0x9999","Name":"xxxx","Contact":0,"Endpoint":1,"LinkQuality":92}}`

And using the test/pairing button:
`{"xxxx":{"Device":"0x9999","Name":"xxxx","ModelId":"lumi.sensor_magnet.aq2","BatteryVoltage":3.05,"BatteryPercentage":100,"Contact":1,"Endpoint":1,"LinkQuality":87}}`

My Tasmota version is 2022.01.3, a development version not much newer than your release version.

Don't know details, but sometimes I've seen improved results redoing the pairing, when the first time seemed to only produce partial results.

**Comment by f45tb00t** [comment]
Hi,

I am on tasmota v10.0.0 with my sonoff bridge.  The bridge is connected via TCP to Home Assistants ZHA integration.

For the include process I pressed long the button on the device until it starts blinking. Then when it started to ask the device I pressed the button like every second.  For one of the devices I needed to repeat it, as it was recognized as unknown device.

Let me know if that helps.

Cheers

fb

**Reply by sfromis** [reply]
When it comes to using ZHA, it ceases to be a question relevant to Tasmota, as this integration facilitates ZHA taking *full* control over the Zigbee chip, with Tasmota only providing a simple ser2net bridge. Whatever happens or not is up to how well ZHA works with the Zigbee chip and connected devices. As long as the serial/TCP communication works, any issues you may have with making a ZHA setup work should be addresses in that project, not Tasmota.

**Comment by f45tb00t** [comment]
Well... From them I got sent here...

**Reply by sfromis** [reply]
That might be due to you, like what you did here, came with your issue without documenting any details, actively skipped answering the questions in the form. You can see how providing details helps the process move along.

**Comment by f45tb00t** [comment]
Aha...

**Comment by namirda** [comment]
So if I understand this correctly, the accepted answer is not an answer at all because the author is using ZHA rather than Z2T.

The original problem therefore remains - Z2T does not currently support the latest door sensors (MCCGQ14LM) from Aqara 

Can you please remove the 'answered' flag on this thread to properly indicate that this issue is still active.

Thanks

**Reply by sfromis** [reply]
Since your situation is different from what the opener of the thread had, it would make sense to open a new thread, instead of trying to reuse a thread for something different.

**Reply by namirda** [reply]
OK - I will open a new thread

Please see new thread [here](https://github.com/arendst/Tasmota/discussions/14543)

**Reply by CrazyMan2000** [reply]
Could you please make a reference to the new thread here? So all other can find it more easily.
